### PR TITLE
chore: json paths

### PR DESCRIPTION
**Pull request by bryce-mcmath**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,15 +37,16 @@ TRACTION_BASE_URL="https://traction-tenant-proxy-dev.apps.silver.devops.gov.bc.c
 TRACTION_WALLET_ID="b1d5b628-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
 TRACTION_WALLET_KEY="286e7818-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
 GOOGLE_AUTH_JSON_PATH="path_to_google_oauth_json_key_from_src.json"
+REQUEST_ATTESTATION_JSON_PATH="fixtures/request_attestation.json"
+OFFER_JSON_PATH="fixtures/offer.json"
```
Comment (jleach) [inline README.md:41]: I think this should just be a path to the message templates. That way we can add more as needed and we don'n need to update the OCP deployment.

**Review by jleach — CHANGES_REQUESTED** [review]
I think we should just use a MESSAGE_TEMPLATE_PATH in the .env and chart then in code we can specify the file to load.

File: README.md (hunk 1 above)
Comment (bryce-mcmath) [reply README.md]: done

File: devops/charts/controller/values.yaml [hunk 2]
```diff
@@ -20,6 +20,7 @@ env:
   TRACTION_BASE_URL: "https://traction-tenant-proxy-dev.apps.silver.devops.gov.bc.ca"
   APPLE_ATTESTATION_ROOT_CA_URL: "https://www.apple.com/certificateauthority/Apple_App_Attestation_Root_CA.pem"
   GOOGLE_AUTH_JSON_PATH: "/opt/controller/google_oauth_key.json"
+  MESSAGE_TEMPLATES_PATH: "opt/fixtures/"
```
Comment (jleach) [inline devops/charts/controller/values.yaml:23]: Needs a starting `/`.

File: src/controller.py [hunk 3]
```diff
@@ -24,10 +28,11 @@ def handle_connection(connection_id):
         print("connection is not completed")
         return
 
-    with open('../fixtures/request_attestation.json', 'r') as f:
+    message_templates_path = os.getenv("MESSAGE_TEMPLATES_PATH")
+    with open(message_templates_path + 'request_attestation.json', 'r') as f:
```
Comment (jleach) [inline src/controller.py:32]: os.path.join(path_part1, path_part2)

File: src/traction.py [hunk 4]
```diff
@@ -97,7 +97,8 @@ def offer_attestation_credential(conn_id):
         "Authorization": f"Bearer {token}"
     }
 
-    with open('../fixtures/offer.json', 'r') as f:
+    message_templates_path = os.getenv("MESSAGE_TEMPLATES_PATH")
+    with open(message_templates_path + 'offer.json', 'r') as f:
```
Comment (jleach) [inline src/traction.py:101]: os.path.join(path_part1, path_part2)

**Review by jleach — CHANGES_REQUESTED** [review]

File: src/controller.py (hunk 3 above)
Comment (bryce-mcmath) [reply src/controller.py]: done

File: devops/charts/controller/values.yaml (hunk 2 above)
Comment (bryce-mcmath) [reply devops/charts/controller/values.yaml]: done

File: src/traction.py (hunk 4 above)
Comment (bryce-mcmath) [reply src/traction.py]: done

**Review by jleach — APPROVED** [review]

**Pull request merged**